content/operations/GOCDB/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "GOCDB"
+description: "GOCDB documentation"
+weight: 20
+type: "docs"
+---
+
+{{% pageinfo %}}
+This documentation about the GOCDB is a work in progress,
+the main source of documentation about GOCDB is still https://wiki.egi.eu/wiki/GOCDB.
+{{% /pageinfo %}}
+
+Grid Configuration Database (GOCDB) contains general information about the
+sites participating to the production Grid. Accessed by all the project actors
```

Comment:
```suggestion
sites participating to the production infrastructure. Accessed by all the project actors
```

Reply:
also project seems not the right word

Reply:
It's the introductotry blurb from the wiki.
We could just say `accessed by all the actors`?
Or a better/more clear proposition?